frontend/src/app/todo/language/language.component.scss

```diff
@@ -0,0 +1,27 @@
+.select-container
+{
+	position: fixed;
+	right: 0;
+	top: 0;
+}
+
+.en
+{
+	background-image: url('https://purecatamphetamine.github.io/country-flag-icons/3x2/GB.svg');
+	background-position: right;
```

Comment:
you repeat yourself 3 times here:

background-position: right;
background-repeat: no-repeat;